content/product-engineering/engineering/index.md

```diff
@@ -83,6 +83,19 @@ Sourcegraph has a lot of repositories!
 - [github.sgdev.org](https://github.sgdev.org) is a Github test instance.
 - [bitbucket.sgdev.org](https://bitbucket.sgdev.org) is a Bitbucket test instance.
 
+## Slack channels
+
+Slack channels for non-team-specific engineering interests typically start with a #dev- prefix
+
+The current channels are:
+
+- #dev-chat
+- #dev-ops
+- #dev-experience
+- #dev-frontend
+- #dev-databases
+- #dev-urandom
+
```

Comment:
```suggestion
- #prod-eng-announce 

```

@daxmc99 do you think this belongs here? I know it throws off the nice prefixing. [Until like two weeks ago](https://sourcegraph.slack.com/archives/C0EPTDE9L/p1637194172100600) this was called #dev-announce, potentially we could see if we want to change it to something like #dev-product-announce?

Reply:
I would say it does!